### PR TITLE
Redux Middleware

### PR DESCRIPTION
**Pull request by CristianRWolox**
## Summary

- Add redux-thunk to the project.

## Trello Card
https://trello.com/c/6YxECpTs/28-n-92-redux-ii-middleware

### CODE REVIEW

File: wbooks/src/redux/book/actions.ts [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
+export const actions = {
+  GET_BOOKS: '@@BOOK/GET_BOOKS',
+  GET_BOOKS_SUCCESS: '@@BOOK/GET_BOOKS_SUCCESS',
+  GET_BOOKS_FAILURE: '@@BOOK/GET_BOOKS_FAILURE'
+};
+
+const actionCreators = {
+  getBooks: () => (dispatch: Dispatch) => {
+    dispatch({ type: actions.GET_BOOKS, isFetching: true });
+    const response = { ok: true, data: BOOKS_MOCK, error: '' };
```
Comment (guidoprinc) [inline wbooks/src/redux/book/actions.ts]: replace with a promise

File: wbooks/src/redux/book/actions.ts [hunk 2]
```diff
@@ -1,6 +1,30 @@
...
+    if (response.ok) {
+      dispatch({
+        type: actions.GET_BOOKS_SUCCESS,
+        isFetching: false,
+        payload: response.data
+      });
+    } else {
+      dispatch({
+        type: actions.GET_BOOKS_FAILURE,
+        isFetching: false,
```
Comment (guidoprinc) [inline wbooks/src/redux/book/actions.ts]: replace `isFetching` with `loading`

**Review by guidoprinc — CHANGES_REQUESTED** [review]

File: wbooks/src/redux/book/actions.ts (hunk 1 above)
Comment (CristianRWolox) [reply wbooks/src/redux/book/actions.ts]: Done!

File: wbooks/src/redux/book/actions.ts (hunk 2 above)
Comment (CristianRWolox) [reply wbooks/src/redux/book/actions.ts]: Done!

**Review by guidoprinc — APPROVED** [review]

**Pull request merged**